nevergrad/optimization/optimizerlib.py

```diff
@@ -2275,6 +2277,35 @@ def __init__(
 NaiveIsoEMNA = EMNA().set_name("NaiveIsoEMNA", register=True)
 
 
+@registry.register
+class modcma(base.Optimizer):
```

Comment:
Please use conventions to name optimizers. That would be ModCMA

Reply:
Fixed.